Documentation/compatibility.md

```diff
@@ -1,9 +1,17 @@
-<br>
-<div class="alert alert-info" role="alert">
-    <i class="fa fa-exclamation-triangle"></i><b> Note:</b> Starting with v0.39.0, Prometheus Operator requires use of Kubernetes v1.16.x and up.
-</div>
-
-# Compatibility
+---
+title: "Compatibility"
+description: "The Prometheus Operator supports a number of Kubernetes and Prometheus releases."
+lead: ""
+date: 2021-03-08T08:49:31+00:00
+lastmod: 2021-03-08T08:49:31+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "operator"
+weight: 200
+toc: true
+---
```

Comment:
Header is encoded in https://github.com/prometheus-operator/prometheus-operator/blob/master/cmd/po-docgen/compatibility.go#L24 and needs to be modified there